lib/id3tag/frames/v2/url_frame.rb

```diff
@@ -0,0 +1,14 @@
+module ID3Tag
+  module Frames
+    module V2
+      class UrlFrame < BasicFrame
+        def url
+          raw_content
```

Comment:
Can't we use original `content` method from BasicFrame? Only thing that could be added would be `url` alias to align the API with UserUrlFrame

Reply:
👍 Done in https://github.com/krists/id3tag/pull/39/commits/1f260254718ba2590c3b05daf5105a1eff6a6a3d